lib/ansible/modules/cloud/google/gcp_compute_firewall_facts.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017 Google
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file at
+#     https://www.github.com/GoogleCloudPlatform/magic-modules
+#
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+################################################################################
+# Documentation
+################################################################################
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: gcp_compute_firewall_facts
+description:
+  - Gather facts for GCP Firewall
+short_description: Gather facts for GCP Firewall
+version_added: 2.7
+author: Google Inc. (@googlecloudplatform)
+requirements:
+    - python >= 2.6
+    - requests >= 2.18.4
+    - google-auth >= 1.3.0
+options:
+    filters:
+       description:
+           A list of filter value pairs. Available filters are listed here
+           U(https://cloud.google.com/sdk/gcloud/reference/topic/filters).
+           Each additional filter in the list will act be added as an AND condition
+           (filter1 and filter2)
+extends_documentation_fragment: gcp
+'''
+
+EXAMPLES = '''
+- name:  a firewall facts
+  gcp_compute_firewall_facts:
+      filters:
+      - name = test_object
```

Comment:
It looks like the return on this is shown as a single item (`{allowed: .... }`) when it should be `{items: [{allowed: .....}, ...]}`